README.md

```diff
@@ -1,3 +1,56 @@
 # Watchdog
 
 A package to provide improved watchdog functionality.
+
+Only works on an actual ESP.
+
+## Usage
+
+Using the watchdog requires to install the provider and then to
+import the client in the applications that want to use it.
+
+### Provider
+
+Either install the `src/provider.toit` as container directly (mainly
```

Comment:
I'd think we'd want Artemis to provide this, so this quickly becomes dated.

Reply:
done.